File: project-euler/Euler002.py
T = int(raw_input())

for t_i in range(T):
    N = int(raw_input())
    
    a = 1
    b = 2
    sum = 0
    
    while (b <= N):
        if b%2 == 0:
            sum += b
        tmp = a
        a = b
        b += tmp
    print sum

File: project-euler/Euler002.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {

    int T;
    scanf("%d",&T);
    
    while (T--) {
        
        long long N;
        scanf("%lld",&N);
        
        long long sum = 0;
        long long a = 1;
        long long b = 2;
        long long tmp;
        
        while (b <= N) {
            if (b % 2 == 0) sum += b;
            
            tmp = a;
            a = b;
            b += tmp;
        }
        
        printf("%lld\n",sum);
        
    }  
    return 0;
}